### PR TITLE
Added homepage.html and relevant css file for homepage

### PR DESCRIPTION
**Pull request by razashtar93**
I created the folder "landing" and in it created the file homepage.html, this is the landing page.
The file only contain a navbar for now

in the folder static/css created the file hompage.css with the necessary css for homepage.html

also added relevant lines in urls.py and views.py for the homepage to be displayed on vagrant up

### CODE REVIEW

**Review by DanWaldman — APPROVED** [review]
LGTM

**Comment by DanielItkes** [comment]
Nice, good job!

**Review by DanielItkes — APPROVED** [review]

File: AcadeMe/urls.py [hunk 1]
```diff
@@ -5,5 +5,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.landingPage,  name='landing_page')
+    # path('', views.landingPage,  name='landing_page'),
```
Comment (Omeramsc) [inline AcadeMe/urls.py:8]: comment not needed, should be removed.

File: AcadeMe/views.py [hunk 2]
```diff
@@ -2,5 +2,8 @@
 # redirect
 
 
-def landingPage(request):
-    return render(request, 'landingpage/landing_page.html')
+# def landingPage(request):
```
Comment (Omeramsc) [inline AcadeMe/views.py:5]: same.

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Remove the meaningless comments, besides that - LGTM. 
(don't worry about losing old code, since everything is on GH you can always reference it if needed. :) )

**Comment by DeanCivlin** [comment]
@Omeramsc We are trying to figure out how I can contribute code changes to this PR. Since we are working together on this feature.
Should I create a PR to Raz branch and then the changes will appear here?
Thanks

**Comment by Yarboa** [comment]
> @Omeramsc We are trying to figure out how I can contribute code changes to this PR. Since we are working together on this feature. Should I create a PR to Raz branch and then the changes will appear here? Thanks

Please review git challenge, Hint: *Part 4: Dealing with cross-PR dependencies*

**Review by Omeramsc — APPROVED** [review]
LGTM

**Review by DeanCivlin — APPROVED** [review]
Good job Raz! I took the code you wrote and as decided between us, extended it and pushed another PR with HomePage changes and App Layout addition. (PR #97)

File: templates/landing/homepage.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                <div>Welcome To AcadeMe.</div>
+            </div>
+        </div>
+    </div>
+    <div class="flex-row bottom-container justify-content-center" style="margin-top: 30px">
+        <div class="flex-col m-5">
+            <h3 class="my-1">Choose University</h3>
+            <div>
+                <select aria-label="choose-university" id="selectContentUniversity">
+                    <option selected>Choose University</option>
```
Comment (Omeramsc) [inline templates/landing/homepage.html:59]: Should not be hardcoded, but dynamic

File: templates/landing/homepage.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                    <option value="1">Reichman University</option>
+                    <option value="2">Hebrew University of Jerusalem</option>
+                    <option value="3">Ben Gurion University</option>
+                    <option value="4">Tel-Aviv University</option>
+                    <option value="5">Technion University</option>
+                </select>
+            </div>
+        </div>
+        <div class="flex-col m-5">
+            <h3 class="my-1">Choose Degree</h3>
```
Comment (Omeramsc) [inline templates/landing/homepage.html:69]: Should not be hardcoded, but dynamic

**Review by Yarboa — COMMENTED** [review]
Please refer #97 comments it is all the same
Lets try to push 97 to be merged first, or implement my comments from there here.
Thanks

**Review by Yarboa — APPROVED** [review]
There are comments, see comments

**Pull request merged**